### PR TITLE
Module complementary hours

### PR DESCRIPTION
**Pull request by nandolm**

### CODE REVIEW

File: src/pages/ComplementaryHours/index.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+import { FiX } from 'react-icons/fi';
+
+import Button from '../../components/Button';
+import ModuleContainer from '../../components/ModuleContainer';
+import Input from '../../components/Input';
+
+import { Container, Table, Modal, ModalContainer, Title } from './styles';
+import Select from '../../components/Select';
+
+const dataActivities = [
```
Comment (KevenMax) [inline src/pages/ComplementaryHours/index.js:11]: Remove the comments.

**Review by KevenMax — CHANGES_REQUESTED** [review]
@nandolm you need to add the link to this module on menu.

File: src/pages/ComplementaryHours/index.js (hunk 1 above)
Comment (nandolm) [reply src/pages/ComplementaryHours/index.js]: Ready

**Review by KevenMax — APPROVED** [review]
Good work!

**Pull request merged**